docs/roles/project-coordinator.rst

```diff
@@ -1,5 +1,4 @@
-Eboard: Project coordinator
-===========================
+Eboard: Project coordinator ===========================
```

Comment:
Looks like this was all accidentally put onto a single line. Add a hard return to the equals sign line so it's rendered correctly.

_Side note_: Not sure why the project built successfully with this…